### DISCUSSION TITLE
Always 2 calls in case of fail or succeed

### DISCUSSION THREAD

**Discussion by marionfromfrance** [spatie/laravel-webhook-server#123] [Q&A]
Hi everybody,

I'm a beginner in webhooks and there is something I don't understand, maybe somebody will be able to help me. When I call my code with WebhookCall::create()->aso, and log into my events listeners, I always see two calls (not 3 as specified in the configuration file). Here are the dumps of event class name and event response status for succeeded call:
^ "Spatie\WebhookServer\Events\WebhookCallSucceededEvent"
^ 200
^ "Spatie\WebhookServer\Events\WebhookCallSucceededEvent"
^ 200
And here are dumps for failed calls:
^ "Spatie\WebhookServer\Events\WebhookCallFailedEvent"
^ 404
^ "Spatie\WebhookServer\Events\WebhookCallFailedEvent"
^ 404

I'm using PHP7 and Laravel 8

Any ideas?

Thank you very much,

Marion

**Comment by marionfromfrance** [comment]
Hi,
looks like it's a laravel problem (I'm using 8 version and all events listeners are fired two times).

**Reply by jonaswebdev** [reply]
I'm facing the same issue on Laravel 11 with laravel-webhook-server 3.8 when implementing the [event subscribers](https://laravel.com/docs/11.x/events#event-subscribers) approach.
